concourse/pipeline.yml

```diff
@@ -213,7 +213,7 @@ jobs:
           CF_APP_NAME: govuk-account-manager
           CF_SPACE: production
           CF_STARTUP_TIMEOUT: 15 # minutes
-          APP_INSTANCES: 20
+          CF_APP_SCALING_POLICY: policy.json
           WORKER_INSTANCES: 20
```

Comment:
Not touched workers so far, we can have a chat about these once  we've got the app scaling perhaps?

Reply:
Do PaaS autoscaling rules only work for the `web` process?  I can't see anything in the docs about scaling other processes.

Reply:
Yeah just took a look around, I can't find any references to this.
Odd, no? Scaling of workers sonds like something that could be quite useful - especially based on metrics around queue length 🤔

Reply:
I've found this: https://github.com/cloudfoundry-samples/cf-autoscaler
but seems like that's a roll-your-own autoscaler for RabiitMQ